.appveyor.ps1

```diff
@@ -0,0 +1,80 @@
+$ErrorActionPreference = "Stop"
+
+function run(){
+    param($command)
+    & $env:ComSpec /c ($command + ' 2>&1')
+}
+
+# consts
+$luatar = 'LuaJIT-2.0.5.zip'
```

Comment:
I specify version because of these reasons:

* How to get the newest version in script? People could look through the website to know the newest version, but it's hard to implement in script. A way is to clone the git repo of luajit, but that's unreleased and riskful
* I need to verify the downloaded file's MD5. If don't specify a version, I could not know its MD5
* lupa seems to fail tests when bundled with lua5.3 by my test

A good way is to accept arguments to let users specify which version to build